src/it/java/com/google/checkstyle/test/base/AbstractGoogleModuleTestSupport.java

```diff
@@ -129,6 +129,52 @@ else if (moduleId == null) {
         return result;
     }
 
+    /**
+     * Returns list of {@link Configuration} instance for the list of module name.
+     * This implementation uses {@link #getModuleById(String)} method inside.
+     *
+     * @param moduleIds list of module id.
+     * @return list of {@link Configuration} instances present inside TreeWalker.
+     * @throws CheckstyleException if there is a problem retrieving the module by id.
+     */
+    protected static List<Configuration>
+        getModuleByIds(List<String> moduleIds) throws CheckstyleException {
```

Comment:
Where is this method used?

I am not seeing it, so I can't say why we need `List` and not `String...`.